runtime/caml/prims.h

```diff
@@ -20,7 +20,7 @@
 
 #ifdef CAML_INTERNALS
 
-typedef value (*c_primitive)();
+typedef value (*c_primitive)(void);
```

Comment:
You could just as well use
```
typedef void * c_primitive;
```
The built-in primitives from table `caml_builtin_cprim` are never called directly anyway, just added to an extensible table with type `void *`.

(Note that ISO C forbids casting a function pointer to type `void *`, but we've been doing this for a very long time, and all C compilers that I know of are happy with this.)

Reply:
Also nice, thanks.